PharoObjectModel/PharoObjectModel.pier

```diff
@@ -292,56 +296,66 @@ Dog class>>>count
     ^ count
 ]]]
 
```

Comment:
But I do not see the Dog 

Dog class subclass: #Object 
     classinstanceVariableName: 'count''

I pulling everything and reading the chapter because this interface is annoying :)